client/src/App.js

```diff
@@ -25,24 +25,28 @@ function App() {
   }, [dispatch]);
   console.log(state);
 
+  // Maybe we should wrap it this way?
+  // if (state.loading) {
+  //   return "Loading";
+  // } else { }
   return (
     <userContext.Provider value={state.user}>
       <MuiThemeProvider theme={theme}>
         <BrowserRouter>
           <Switch>
             <Route path="/signup" component={SignUp} />
             <Route path="/login" component={Login} />
-            <ProtectedRoute condition={() => state.user.id} path="/onboard">
+            <ProtectedRoute condition={() => state.user._id} path="/onboard">
               <OnBoard state={state} dispatch={dispatch} />
             </ProtectedRoute>
             <Route path="/" exact component={SignUp} />
-            <ProtectedRoute condition={() => state.user.id} path="/profile">
+            <ProtectedRoute condition={() => state.user._id} path="/profile">
               <Profile state={state} dispatch={dispatch} />
             </ProtectedRoute>
-            <ProtectedRoute condition={() => state.user.id} path="/reviews">
+            <ProtectedRoute condition={() => state.user._id} path="/reviews">
               <Reviews state={state} dispatch={dispatch} />
             </ProtectedRoute>
-            <ProtectedRoute condition={() => state.user.id} path="/balance">
+            <ProtectedRoute condition={() => state.user._id} path="/balance">
```

Comment:
In #48 
- loadingContext provides for `const loading = useContext(loadingContext);`
- transform on `User.prototype.toObject` that formats `_id` as `id`